inputremapper/injection/consumers/mapping_handler.py

```diff
@@ -504,3 +497,163 @@ async def notify(
             forward=forward,
             supress=supress,
             )
+
+
+class AbsToRelHandler:
+    """
+    Handler which transforms an EV_ABS to EV_REL events
+    and sends that to a UInput
+
+    adheres to the MappingHandler protocol
+    """
+    _key: Key  # key of len 1 for the event to
+    _target: str  # name of target UInput
+    _deadzone: float  # deadzone
+    _output: int  # target event code
+
+    # the ratio between abs value as float between -1 and +1
+    # and the output speed as units per tick
+    _gain: float
+    _expo: float
+    _rate: int  # the tick rate in Hz
+
+    _last_value: float  # value of last abs event between -1 and 1
+    _running: bool  # if the run method is active
+    _stop: bool  # if the run loop should return
+
+    def __init__(self, config: Dict[str, any]) -> None:
+        """initialize the handler
+
+        Parameters
+        ----------
+        config : Dict = {
+            "key": str
+            "output": int
+            "target": str
+            "deadzone" : int
+            "output" : int
+            "gain" : float
+            "rate" : int
+        }
+        """
+        self._key = Key(config["key"])
+        self._target = config["target"]
+        self._deadzone = config["deadzone"] / 100
+        self._output = config["output"]
+        self._gain = config["gain"]
+        self._rate = config["rate"]
+
+        self._last_value = 0
+        self._running = False
+        self._stop = True
+
+    async def notify(
+            self,
+            event: evdev.InputEvent,
+            source: evdev.InputDevice = None,
+            forward: evdev.UInput = None,
+            supress: bool = False,
+            ) -> bool:
+
+        if (event.type, event.code) != self._key[0][:2]:
+            return False
+
+        input_value, scale_factor = self._normalize(
+            event.value,
+            source.absinfo(event.code).min,
+            source.absinfo(event.code).max,
+            )
+        if input_value < self._deadzone:
+            self._stop = True
+            return True
+
+        output_value = self._calc_qubic(input_value, self._expo)
+        self._last_value = output_value * scale_factor * self._gain
+
+        if not self._running:
+            asyncio.ensure_future(self._run())
+        return True
+
+    @staticmethod
+    def _calc_qubic(x: float, k: float) -> float:
```

Comment:
is this for making the cursor move faster when the abs value is at its maximum?

Reply:
Yes, have a look at this:
https://www.geogebra.org/calculator/mkdqueky

Reply:
cool, can we have this link in the docstring?

Reply:
I am not sure how long lived that is. But sure I will add it